Formula/envoy.rb

```diff
@@ -28,8 +28,8 @@ def install
       --curses=no
       --show_task_finish
       --verbose_failures
-      --action_env=PATH=/usr/local/bin:/opt/local/bin:/usr/bin:/bin
-      --test_output=all
+      --action_env=PATH=#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin
```

Comment:
You need to change the `%w` to a `%W`, or else `HOMEBREW_PREFIX` won't get interpolated.

Reply:
sure I need to reorganize the commit anyway

Reply:
ok got this in. next step is to see how best to delete a line before building